cypress/integration/patch-route.js

```diff
@@ -0,0 +1,35 @@
+/// <reference types="Cypress" />
+
+const patch = {
+  id: "5983798397b1d35eb800040f",
+  desc: "Ernie testing 20"
+};
+
+const badPatch = {
+  id: "i-dont-exist"
+};
+
+describe("Patch route", function() {
+  beforeEach(() => {
+    cy.login();
+  });
+
+  it("Loads patch data and renders it on the page", function() {
+    cy.visit(`/patch/${patch.id}`);
+    cy.get("h1[id=patch-name]").should("include.text", patch.desc);
+  });
+
+  it("Clicking 'Base commit' link in metadata goes to version page of legacy UI", function() {
+    cy.visit(`/patch/${patch.id}`);
+    cy.get("a[id=patch-base-commit]").click();
+    cy.url().should(
```

Comment:
This test can be more robust if we replace this line to match the pathname only
 cy.location('pathname').should('eq', `https://evergreen.mongodb.com/version/${patch.id}`)

Reply:
sure